scripts/env/dev/init.sh

```diff
@@ -103,6 +110,8 @@ else
               ;;
           s)
              source ${S3_SRC_DIR}/scripts/env/common/setup-yum-repos.sh
+             # install pip3 package toml if not present, needed by cortx-py-utils for s3confstore UT
```

Comment:
Please update same changes to scripts/env/release/init.sh and scripts/env/rpmbuild/init.sh as well